packages/rpc-transport/src/json-rpc-types.ts

```diff
@@ -54,7 +54,7 @@ type PendingRpcRequestBuilder<TMethodImplementations> = UnionToIntersection<
 type Callable = (...args: any[]) => any;
 type Flatten<T> = T extends (infer Item)[] ? Item : never;
 type Overloads<T> =
-    // Have an RPC method with more than 10 overloads? Add another section and update this comment
+    // Have an RPC method with more than 25 overloads? Add another section and update this comment
```

Comment:
![work-out-excercise](https://github.com/solana-labs/solana-web3.js/assets/13243/cdb5f7c8-e4da-489b-8f0a-268418af0058)

Reply:
It hurt to change that comment